confluent_server/confluent/messages.py

```diff
@@ -634,6 +643,58 @@ def __init__(self, path, nodes, inputdata):
     def domain_name(self, node):
         return self.inputbynode[node]
 
+
+class InputNTPServer(ConfluentInputMessage):
+    def __init__(self, path, nodes, inputdata):
+        self.inputbynode = {}
+        self.stripped = False
+        if not inputdata or 'server' not in inputdata:
+            raise exc.InvalidArgumentException('missing input data')
+        if len(inputdata['server']) > 256:
+            raise exc.InvalidArgumentException(
+                'identifier must be less than or = 256 chars')
+
+        if nodes is None:
+            raise exc.InvalidArgumentException(
+                'This only supports per-node input')
+        for node in nodes:
+            self.inputbynode[node] = inputdata['server']
+
+    def ntp_server(self, node):
+        return self.inputbynode[node]
+
+class InputNTPServers(ConfluentInputMessage):
+    def __init__(self, path, nodes, inputdata):
+        self.inputbynode = {}
+        self.stripped = False
+        if not inputdata:
+            raise exc.InvalidArgumentException('missing input data')
+
```

Comment:
While I'm glad it's in an array, I think I'd like to see the array under a key name, e.g. 'ntp_servers' : []